ci/tasks/deploy-test-services.yml

```diff
@@ -10,6 +10,7 @@ params:
   DEPLOYER_ROLE_ARN: ((deployer-role-arn-non-prod))
   DEPLOY_ENVIRONMENT: build
   STATE_BUCKET: digital-identity-dev-tfstate
+  STATE_LOCKING_TABLE: digital-identity-dev-tfstate-locking
```

Comment:
this task has it set?

Reply:
That's a "whoops I didn't save it in my last PR" 😅